packages/pluggableWidgets/safe-area-view-native/package.json

```diff
@@ -20,7 +20,8 @@
   },
   "dependencies": {
     "@mendix/piw-native-utils-internal": "*",
-    "@mendix/piw-utils-internal": "*"
+    "@mendix/piw-utils-internal": "*",
+    "react-native-safe-area-context": "4.11.0"
```

Comment:
I see that we're using rn-safe-area-context version 5.2.0 in both the Native Template and MiN, so this widget should use the same version for consistency.

https://github.com/mendix/native-template/blob/master/package.json#L35
https://github.com/mendix/make-it-native/blob/main/package.json#L65